tests/scss/base/_base-theme.scss

```diff
@@ -0,0 +1,11 @@
+/*  Base Theme  */
```

Comment:
This is all pretty great. I wanted this for a bit but opened external projects for the longest time. I could even potentially write some automated tests using these fixtures. 

Many thanks!